web/src/app/main/components/NavSubMenu.js

```diff
@@ -41,6 +41,9 @@ const useStyles = makeStyles(theme => {
     dropdownClosed: {
       transform: 'rotate(-90deg)',
     },
+    listItemText: {
+      color: theme.palette.primary['500'],
```

Comment:
I think this should use `main` (should be the same as 500, but more obvious we want the "default")
```suggestion
      color: theme.palette.primary.main,
```